soc/nordic/nrf54h/Kconfig

```diff
@@ -28,7 +28,6 @@ config SOC_NRF54H20_CPUAPP_COMMON
 	select HAS_SEGGER_RTT if ZEPHYR_SEGGER_MODULE
 	select NRFS_HAS_AUDIOPLL_SERVICE
 	select NRFS_HAS_CLOCK_SERVICE
-	select NRFS_HAS_DVFS_SERVICE
```

Comment:
remove this, with https://github.com/zephyrproject-rtos/zephyr/pull/91187 it will not be needed :)